### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Command/CreateCommand.php [hunk 1]
```diff
@@ -30,11 +30,11 @@
  *
  * This command creates a new migration using the available migration template.
  *
- * Config in di-container console.php migrations paths `createPath` and `updatePaths`:
+ * To use it, configure migrations paths (`createPath` and `updatePaths`) in DI container via `console.php`:
```
Comment (samdark) [inline src/Command/CreateCommand.php:33]: That looks like too much for command phpdoc. Could it be moved to docs?

File: src/Command/DownCommand.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
  * yii migrate:down all # revert all migrations
  * ```
  */
-#[AsCommand('migrate:down', 'Downgrades the application by reverting old migrations.')]
+#[AsCommand('migrate:down', 'Reverts the specified number of latest migrations.')]
```
Comment (samdark) [inline src/Command/DownCommand.php:34]: I have changed strings but did not adjust tests.

**Review by terabytesoftw — APPROVED** [review]

File: src/Command/DownCommand.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Command/DownCommand.php:34]: done.

File: src/Command/CreateCommand.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Command/CreateCommand.php]: fixed.

**Pull request merged**